stories/SQForm.stories.tsx

```diff
@@ -400,7 +401,9 @@ export const formWithInclusionlist = () => {
             }
           }
         >
-          {(arrayHelpers) => {
+          {(
+            arrayHelpers: FieldArrayRenderProps
+          ): JSX.Element | Array<JSX.Element> => {
```

Comment:
This function is actually always returning just `JSX.Element`, right? One element and it's a grid.

Reply:
+1

Reply:
Correct.  Just changed it. Thanks.